gnome/gtk-vnc/Portfile

```diff
@@ -82,3 +99,6 @@ if {![variant_isset quartz] && ![variant_isset x11]} {
         return -code error "Either +x11 or +quartz is required"
     }
 }
+
+livecheck.type      gnome
+livecheck.regex     LATEST-IS-(\\d+\\.\\d*\\d(?:\\.\\d+)*)
```

Comment:
> Why override `livecheck.regex`? Shouldn't setting `livecheck.type gnome` be sufficient? The reason for having livecheck types at all is to consolidate knowledge about a particular hosting system's rules in one place rather than spread out amongst hundreds of portfiles.
> 
> If you're doing this because `livecheck.type gnome` does the old even-is-stable thing and you want all versions, not just even ones, then you could use `livecheck.type gnome-with-unstable`.
> 
> These livecheck types probably need to be renamed since gnome stopped doing the even-is-stable thing in 2020.

We can certainly fix this as well, with the caveat that I was trying to avoid changing too much. (As it is, there are still ample changes anyway.)

Reply:
>> Why override `livecheck.regex`? Shouldn't setting `livecheck.type gnome` be sufficient?

> We can certainly fix this as well, with the caveat that I was trying to avoid changing too much. (As it is, there are still ample changes anyway.)

Okay, so the reason for the custom regex, is that `gtk-vnc` releases can be odd-numbered. So the easy solution is to change `livecheck.type` to `gnome-with-unstable`.